lagunita/lms/static/sass/_overrides.scss

```diff
@@ -224,116 +224,118 @@ body.discussion {
         display: none;
     }
```

Comment:
I left this stuff outside since it's not within the HTML component to begin with...